lib/web_ui/test/keyboard_test.dart

```diff
@@ -226,13 +235,23 @@ void testMain() {
       expect(count, 2);
     });
 
-    test('prevents default when "Tab" is pressed', () {
+    test('prevents default when key is handled by the framework', () {
+      ByteData _toByteData(List<int> bytes) {
+        final ByteData byteData = ByteData(bytes.length);
+        for (int i = 0; i < bytes.length; i++) {
+          byteData.setUint8(i, bytes[i]);
+        }
+        return byteData;
+      }
```

Comment:
Is it possible to avoid this if you use `JSONMessageCodec.encodeMessage()`?

Reply:
Yes.  Done.